service/src/cli/utils.ts

```diff
@@ -0,0 +1,104 @@
+import columnify from "columnify";
+import commandLineArgs from "command-line-args";
+import { OnchainProfile } from "../utils/onchain-data";
+import { logCallout } from "../logger/formatter";
+import { archLogger } from "../logger/chalk-theme";
+import { formatEther } from "ethers/lib/utils";
+import { exit } from "process";
+import { BigNumber, ethers } from "ethers";
+
+export const handleCommandArgs = (
+  optionDefinitions: any,
+  options: any,
+  commandName: string
+): any => {
+  try {
+    return commandLineArgs(optionDefinitions, options);
+  } catch (err) {
+    logCallout(() => {
+      const errorName = () => {
+        switch (err.name) {
+          case "UNKNOWN_OPTION":
+            return `You used an invalid option: ${err.optionName}`;
+          case "UNKNOWN_VALUE":
+            return `You used an invalid value for one of the options: ${err.value}`;
+          case "ALREADY_SET":
+            return `A value for option ${err.optionName} was already set.}`;
+        }
```

Comment:
These are errors thrown by the `commandLineArgs` package -- I checked to see if there was error codes or an `enum` we could check against, but there isn't, so checked against the error string.